pycolmap/pycolmap/estimators/alignment.h

```diff
@@ -0,0 +1,144 @@
+         const double min_inlier_observations,
+         const double max_reproj_error) {
+        THROW_CHECK_GE(min_inlier_observations, 0.0);
+        THROW_CHECK_LE(min_inlier_observations, 1.0);
+        Sim3d tgt_from_src;
+        THROW_CHECK(
+            AlignReconstructionsViaReprojections(src_reconstruction,
+                                                 tgt_reconstruction,
+                                                 min_inlier_observations,
+                                                 max_reproj_error,
+                                                 &tgt_from_src));
+        return tgt_from_src;
+      },
+      "src_reconstruction"_a,
+      "tgt_reconstruction"_a,
+      "min_inlier_observations"_a = 0.3,
+      "max_reproj_error"_a = 8.0);
+
+  m.def(
+      "align_reconstructions_via_proj_centers",
+      [](const Reconstruction& src_reconstruction,
+         const Reconstruction& tgt_reconstruction,
+         const double max_proj_center_error) {
+        THROW_CHECK_GT(max_proj_center_error, 0.0);
+        Sim3d tgt_from_src;
+        THROW_CHECK(AlignReconstructionsViaProjCenters(src_reconstruction,
+                                                       tgt_reconstruction,
+                                                       max_proj_center_error,
+                                                       &tgt_from_src));
+        return tgt_from_src;
+      },
+      "src_reconstruction"_a,
+      "tgt_reconstruction"_a,
+      "max_proj_center_error"_a);
+
+  m.def(
+      "align_reconstructions_via_points",
+      [](const Reconstruction& src_reconstruction,
+         const Reconstruction& tgt_reconstruction,
+         const size_t min_common_observations,
+         const double max_error,
+         const double min_inlier_ratio) {
+        THROW_CHECK_GT(min_common_observations, 0);
+        THROW_CHECK_GT(max_error, 0.0);
+        THROW_CHECK_GE(min_inlier_ratio, 0.0);
+        THROW_CHECK_LE(min_inlier_ratio, 1.0);
+        Sim3d tgt_from_src;
+        THROW_CHECK(AlignReconstructionsViaPoints(src_reconstruction,
+                                                  tgt_reconstruction,
+                                                  min_common_observations,
+                                                  max_error,
+                                                  min_inlier_ratio,
+                                                  &tgt_from_src));
+        return tgt_from_src;
+      },
+      "src_reconstruction"_a,
+      "tgt_reconstruction"_a,
+      "min_common_observations"_a = 3,
+      "max_error"_a = 0.005,
+      "min_inlier_ratio"_a = 0.9);
+
+  m.def(
+      "align_reconstrution_to_locations",
+      [](const Reconstruction& src,
+         const std::vector<std::string>& image_names,
+         const std::vector<Eigen::Vector3d>& locations,
+         const int min_common_images,
+         const RANSACOptions& ransac_options) {
+        THROW_CHECK_GE(min_common_images, 3);
```

Comment:
I guess we still want to unify CHECKs to be exceptions also in COLMAP?

Reply:
Not for this PR, just aligning on remaining steps.

Reply:
Yes, will address in a later PR